src/MyTested.AspNetCore.Mvc.Caching/ComponentBuilderDistributedCacheWithoutExtensions.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace MyTested.AspNetCore.Mvc
+{
+    using System;
+    using System.Collections.Generic;
+    using MyTested.AspNetCore.Mvc.Builders.Base;
```

Comment:
These usings should use short syntax - 'Builders.Base`. It is the style of the project.